### ISSUE TITLE
Lint/DefEndAllignment Problem.

### ISSUE THREAD

**Issue by anik3tra0** [rubocop-hq/rubocop#4131]
I am using Atom as my Text Editor.
 
![screen shot 2017-03-16 at 10 52 42 am](https://cloud.githubusercontent.com/assets/13035908/23983051/a38d67e2-0a37-11e7-983a-cddc405c024f.png)

(Above is with Inconsistent Indentation warnings)

Above is what I expect when I auto-correct and Below is what I get which I believe is incorrect.

![screen shot 2017-03-16 at 10 53 12 am](https://cloud.githubusercontent.com/assets/13035908/23983074/cd0214ba-0a37-11e7-81ee-1e64f5fac6bd.png)

Is there any config that I am missing?

**Comment by backus** [comment]
Yeah you need to set AutoCorrect to true. See past issues for more detail https://github.com/bbatsov/rubocop/issues?utf8=%E2%9C%93&q=Lint%2FEndAlignment%20

**Comment by anik3tra0** [comment]
Setting autocorrect to true doesn't solve this.

**Comment by jonas054** [comment]
There must be something above these method definitions with a different indentation that causes auto-correct to shift some lines. I would think think that @backus is right about setting
```yaml
Lint/DefEndAlignment:
  AutoCorrect: true
```
@anik3tra0 You said that "setting autocorrect to true" doesn't work. Your use of lower case suggests to me that you're not making the change in the right place.

**Comment by mikegee** [comment]
I suppose @anik3tra0 has resolved or worked around this by now. Should we close this one now?
